docs/supportedsites.md

```diff
@@ -582,6 +583,7 @@
  - **tvp.pl**
  - **tvp.pl:Series**
  - **TVPlay**: TV3Play and related services
+ - **tvple**
```

Comment:
This file is updated automatically in releases and should not be changed in normal commits.